types/aws-lambda/trigger/connect-contact-flow.d.ts

```diff
@@ -0,0 +1,47 @@
+import { Callback, Handler } from '../handler';
+
+export type ConnectContactFlowHandler = Handler<ConnectContactFlowEvent, ConnectContactFlowResult>;
+export type ConnectContactFlowCallback = Callback<ConnectContactFlowResult>;
+
+// Connect
+// https://docs.aws.amazon.com/connect/latest/adminguide/connect-lambda-functions.html
+export interface ConnectContactFlowEvent {
+    Details: {
+        ContactData: {
+            Attributes: { [key: string]: string };
+            Channel: ConnectContactFlowChannel;
+            ContactId: string;
+            CustomerEndpoint: ConnectContactFlowEndpoint | null;
+            InitialContactId: string;
+            InitiationMethod: ConnectContactFlowInitiationMethod;
+            InstanceARN: string;
+            PreviousContactId: string;
+            Queue: string | null;
+            SystemEndpoint: ConnectContactFlowEndpoint | null;
+            MediaStreams: {
+                Customer: {
+                    Audio: {
+                        StartFragmentNumber?: string;
+                        StartTimestamp?: string;
+                        StreamARN?: string;
```

Comment:
Just wondering if these are definitely sometimes not present, while the above are instead sometimes `null`? AWS is oftew pretty weird about these things.

Reply:
Yeah, the AWS documentation is pretty poor for Amazon Connect Lambda triggers, but from all my testing those attributes are definitely sometimes not present, but the other ones are present with null values

Reply:
Yeah, pretty standard stuff from them in both counts. Thanks for verifying this!